README.md

```diff
@@ -80,7 +80,7 @@ Format of the release tags are `<package-name>_<package-version>`. For more info
 
 ## Contributing
 
-For details on contributing to this repository, see the [contributing guide](CONTRIBUTING.md).
+For details on contributing to this repository, see the [contributing guide](https://github.com/Azure/azure-sdk-for-java/blob/master/CONTRIBUTING.md).
```

Comment:
I don't think this is the best way to fix these links. We should try to avoid links to master. Instead we should run the same script we run on the other README's that turn relative links into absolute links as part of the publishing step.

Reply:
Those style of links are all gone with the latest PR.